src/components/ec-blog-item/ec-blog-item.spec.js

```diff
@@ -5,14 +5,33 @@ describe('EcBlogItem', () => {
   function mountEcBlogItem(props, mountOpts) {
     return mount(EcBlogItem, {
       propsData: {
+        title: 'title',
```

Comment:
If you pass the props here, then you don't need to pass the props on the tests, and you can keep only one to test all the props instead of one by property.

Just keep one tests without passing any prop when you are calling the function. Because you are already passing those on the function itself.